assets/i18n/strings_km.i18n.json

```diff
@@ -0,0 +1,415 @@
+  },
+  "sendPage": {
+    "waiting": "កំពុងរង់ចាំការឆ្លើយតប...",
+    "rejected": "អ្នកទទួលបានបដិសេធសំណើនេះ។",
+    "busy": "អ្នកទទួលកំពុងជាប់រវល់ជាមួយសំណើផ្សេងទៀត។"
+  },
+  "progressPage": {
+    "titleSending": "កំពុងផ្ញើឯកសារ",
+    "titleReceiving": "កំពុងទទួលឯកសារ",
+    "savedToGallery": "បានរក្សាទុកក្នុងកម្មវិធីរូបថត",
+    "total": {
+      "title": {
+        "sending": "រយៈពេលប្រតិបត្តិការសរុប ({time})",
+        "finishedError": "បានបញ្ចប់ជាមួយនឹងបញ្ហា",
+        "canceledSender": "ត្រូវបានបោះបង់ដោយអ្នកផ្ញើ",
+        "canceledReceiver": "ត្រូវបានបោះបង់ដោយអ្នកទទួល"
+      },
+      "count": "ឯកសារ: {curr} / {n}",
+      "size": "ទំហំ: {curr} / {n}",
+      "speed": "ល្បឿន: {speed}/s"
+    }
+  },
+  "webSharePage": {
+    "title": "ចែករំលែកតាមតំណភ្ជាប់",
+    "loading": "កំពុងចាប់ផ្តើម server...",
+    "stopping": "កំពុងបញ្ឈប់ server...",
+    "error": "មានកំហុសមួយបានកើតឡើងអំឡុងពេលចាប់ផ្តើម server ។",
+    "openLink": {
+      "one": "បើកតំណនេះនៅក្នុងកម្មវិធី browser:",
+      "other": "បើកតំណមួយក្នុងចំណោមតំណភ្ជាប់ទាំងនេះក្នុងកម្មវិធី browser:"
+    },
+    "requests": "ការស្នើសុំ",
+    "noRequests": "មិនទាន់មានសំណើនៅឡើយ។",
+    "encryption": "@:settingsTab.network.encryption",
+    "encryptionHint": "LocalSend ប្រើវិញ្ញាបនបត្រដែលចុះហត្ថលេខាដោយខ្លួនឯង។ អ្នកត្រូវទទួលយកវានៅក្នុងកម្មវិធី​ browser ។",
+    "pendingRequests": "សំណើដែលកំពុងរង់ចាំ: {n}"
+  },
+  "aboutPage": {
+    "title": "អំពី LocalSend"
+  },
+  "changelogPage": {
+    "title": "កំណត់ត្រាការផ្លាស់ប្តូរ"
+  },
+  "aliasGenerator(ignoreMissing)": {
+    "@info": "មូលដ្ឋានផ្សេងគ្នាអាចមានពាក្យផ្សេងគ្នា វាប្រហែលជាមិនត្រូវគ្នានឹង 1:1 ទេ។",
+    "adjectives": [
+      "គួរអោយស្រលាញ់",
+      "ស្រស់ស្អាត",
+      "ធំ",
+      "ភ្លឺចែកចាំង",
+      "ស្អាត",
+      "ពូកែ",
+      "ឡូយ",
+      "គួរអោយស្រលាញ់",
+      "គួរអោយទាក់ទាញ",
+      "ហ្មត់ចត់",
+      "មានថាមពល",
+      "សន្សំសំចៃ",
+      "អស្ចារ្យ",
+      "លឿន",
+      "ល្អ",
+      "ស្រស់ស្រាយ",
+      "ល្អ",
+      "ស្រស់ស្អាត",
+      "ល្អណាស់",
+      "សង្ហា",
+      "ទាក់ទាញ",
+      "រួសរាយ",
+      "គួរអោយស្រលាញ់",
+      "អាថ៌កំបាំង",
+      "គួរអោយសរសើរ",
+      "ល្អ",
+      "អត់ធ្មត់",
+      "ស្រស់ស្អាត",
+      "មានអំណាច",
+      "អ្នកមាន",
+      "សម្ងាត់",
+      "ឆ្លាតវៃ",
+      "រឹងមាំ",
+      "ពិសេស",
+      "យុទ្ធសាស្ត្រ",
+      "រឹងប៉ឹង",
+      "ផ្ជិតផ្ចង់",
+      "ប្រាជ្ញា"
+    ],
+    "fruits": [
+      "ផ្លែប៉ោម",
+      "ផ្លែបឺរ",
+      "ផ្លែចេក",
+      "ផ្លែ Blackberry",
+      "ផ្លែ Blueberry",
+      "ផ្កាខាត់ណាខៀវ",
+      "ផ្លែការ៉ុត",
+      "ផ្លែឆឺរី",
+      "ផ្លែដូង",
+      "ផ្លែទំពាំងបាយជូ",
+      "ផ្លែក្រូចឆ្មា",
+      "បន្លែ",
+      "ផ្លែស្វាយ",
+      "ផ្លែឪឡឹក",
+      "ផ្សិត",
+      "ខ្ទឹមបារាំង",
+      "ផ្លែក្រូច",
+      "ផ្លែល្ហុង",
+      "ផ្លែប៉េស",
+      "ផ្លែសឺរី",
+      "ផ្លែម្នាស់",
+      "ដំឡូង",
+      "ផ្លែល្ពៅ",
+      "ផ្លែ Raspberry",
+      "ផ្លែស្ត្របឺរី",
+      "ផ្លែប៉េងប៉ោះ"
+    ],
+    "combination": "{adjective} {fruit}",
```

Comment:
Can we just combine the adjective and fruit? In some languages, the adjective must have a specific ending depending on the fruit.

Reply:
Yes, we can! In Khmer language, adjectives are after nouns.